packages/terra-demographics-banner/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Added
```

Comment:
Nit : I feel this should be labelled under `fixed`

Reply:
Addressed here : https://github.com/cerner/terra-core/pull/2865/commits/c6ac544c59a8d66c04554117949998099eca28c0